bin/install-qa-check.d/60pkgconfig

```diff
@@ -1,8 +1,14 @@
 # Check for pkg-config file issues
 
 pkgconfig_check() {
+	# Make a list of .pc files and bail out if there aren't any
+	local files=$(find "${ED}"/usr/lib*/pkgconfig -name '*.pc' -print 2>/dev/null)
```

Comment:
Why u no `-print0` and `read`, and bash array? Also, do we really expect files in subdirs there? If pkg-config doesn't recur into subdirs, neither should we.

Reply:
this one should be safer indeed, `-d` arg is double singlequote.
mapfile -d '' files < <(find "${ED}"/usr/lib*/pkgconfig -maxdepth 1 -type f -name '*.pc' -print0 2>/dev/null)

we need to search `/usr/share/pkgconfig` too, it contains pc files for arch-independent things (headers, icons, etc)

Reply:
Oh, nice.